src/graphs.ts

```diff
@@ -1,120 +1,207 @@
-import slugify from '@sindresorhus/slugify'
-import {ensureDir, ensureFile, writeFile} from 'fs-extra'
-import {join} from 'path'
 import {getConfig} from './helpers/config'
-import {infoErrorLogger} from './helpers/log'
-import {getHistoryItems} from './helpers/calculate-response-time'
-import dayjs from 'dayjs'
-import {ChartJSNodeCanvas} from 'chartjs-node-canvas'
-import {cli} from 'cli-ux'
+import {readFile} from 'fs-extra'
+import {join} from 'path'
+import cli from 'cli-ux'
 import chalk from 'chalk'
 
-const canvasRenderService = new ChartJSNodeCanvas({width: 600, height: 400})
-const chartOptions = {
-  plugins: {
-    legend: {display: false},
-  },
-  scales: {
-    xAxes: {
-      display: false,
-      grid: {
-        display: false,
-      },
-    },
-    yAxes: {
-      display: false,
-      grid: {
-        display: false,
-      },
-    },
-  },
+// Error.stackTraceLimit = Infinity;
+export const Dayvalues = async(slug = "") =>{
+    try{
+        const config = await getConfig()
+        const exists = config.sites.map(ob => ob.name === `${slug}` ?  true : false)
```

Comment:
Double check. ```ob.name``` may not be the slug (Did you mean ```ob.slug || slugify(ob.name)```?). Correct semantics, if meant site-name. But best way is to use slugs.